Source/VCGeneration/Checker.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.Contracts;
+using System.Linq;
```

Comment:
Why is this import necessary since nothing else changed in the file?

Reply:
It's not. Removed it.